tests/files/test_file_0001.txt

```diff
@@ -0,0 +1,3 @@
+This is a test file.
+It contains some text, numbers, and special characters.
+1234567890 !@#$%^&*()
```

Comment:
I know this might seem like a joke, but could you add a unicode emoji to this please? I do actually use them in some places, so we may as well test them! Maybe a 🙂 or something?